Integrations/build.gradle

```diff
@@ -44,9 +46,17 @@ sourceSets {
 JavaPluginConvention java = project.convention.plugins.get('java') as JavaPluginConvention
 SourceSet test = java.sourceSets.maybeCreate('test')
 
-def runInDocker = { String name, String sourcePath, List<String> command, Closure addConfig = {} ->
+def runInDocker = { String name, String sourcePath, def command, Closure addConfig = {} ->
     Docker.registerDockerTask(project, name) {
+        final String JPY_SOURCE_PATH = project.hasProperty("customJpy") ? project.property("customJpy") : null
```

Comment:
Consider moving this to docker/server-jetty/build.gradle, which is upstream of not just this task, but also the client integration tests, and `prepareCompose` itself (allowing devs to test the server directly, not just run through this specific set of tests). Putting it at the end of that set of wiring would make it late in the build (but perhaps not as late as you want, since Integrations will add more stuff after it, though other client integration tests use that image as-is).